source/releasenotes/2024-11-14-solr-module-8-1-x.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Search API Pantheon 8.1.x Release
+published_date: "2024-11-14"
+categories: [drupal, action-required]
+---
+
+## Critical Fix: Solr Core Reload Functionality
+
+This release addresses a critical issue where Solr cores were not automatically reloading after schema updates, leading to schema reversions, search index corruption, and potential site downtime. The update implements automatic core reloading to maintain search index integrity.
+
+### Impact of Previous Behavior
+
+* Unexpected schema reversions causing search functionality to break
+* Search index corruption requiring manual intervention
+* Site downtime due to failed search integrations
+* Loss of custom schema configurations
+* Potential data loss in search indexes
```

Comment:
"data loss" might be an over statement when the data is canonical stored in the database.